pkg/provider/filters.go

```diff
@@ -183,3 +159,28 @@ func (env *TestEnvironment) GetPodsUsingSRIOV() ([]*Pod, error) {
 	}
 	return filteredPods, nil
 }
+
+func getContainers(pods []*Pod) []*Container {
+	var containers []*Container
+
+	for _, pod := range pods {
+		containers = append(containers, pod.Containers...)
+	}
+	return containers
+}
+
+func (env *TestEnvironment) GetGuaranteedPodContainersWithExlusiveCPUs() []*Container {
+	return getContainers(env.GetGuaranteedPodsWithExclusiveCPUs())
```

Comment:
why not use directly these lines to get the containers in the test? looks more readable to me.

Reply:
Since this is the utility function, thought of keeping it here for better understandability.